SU-SWS/decanter-react
# ADAPT-1178 ADAPT-1785 ADAPT-1789: Alert Component and Misc Storybook Config

# READY FOR REVIEW

# Summary
- Alert & Button Component
- Storybook configuration and additional add ons.
- Some default HTML elements added as stories.
- Beginning of jest tests

# Needed By (Date)
- End of the sprint

# Urgency
- Normal
 
# Steps to Test

1. Check out the preview build and try out all of the different control options on the alert component
2. Compare the layouts in the Figma mock to the options provided and record any gaps. 
3. Review styles and responsive behavior for a couple of configurations of levers
4. Review code 
5. Review accessibility / Browser Compatibility
6. Review the scripts section in package.json and run each one to get familiar. 

# Affected Projects or Products
- Decanter, ADAPT

# Associated Issues and/or People
- ADAPT-1178, ADAPT-1785, ADAPT-1789
- @katrialesser 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
